### PR TITLE
50 speakers in place of 10  base_session_table.html #3073 

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
10 speakers are to be changed to changed here also .This changed to 50.

### CODE REVIEW

**Comment by niranjan94** [comment]
@Himanshi-Khandelwal please give a proper commit message. Just `	Update base_session_table.html` doesn't really say what was done.

File: app/templates/gentelella/users/events/sessions/base_session_table.html [hunk 1]
```diff
@@ -233,7 +233,7 @@ <h4 class="modal-title" id="mailModalLabel">Customize
             });
 
         var table = $('.with-datatable').DataTable({
-            "dom": '<"row"<"toolbar col-md-7"<"pull-left"l>><"col-md-3 pull-right"f>>tip',
+            "dom": '<"row"<"toolbar col-md-7"<"pull-left"l>><"col-md-3 pull-right"f>>tip', "pageLength": 50,
```
Comment (SaptakS) [inline app/templates/gentelella/users/events/sessions/base_session_table.html]: Please put this `"pageLength": 50,` to follow the indentation properly. Thanks.

Comment (sudheesh001) [inline app/templates/gentelella/users/events/sessions/base_session_table.html]: This looks like a really really dirty fix.

Comment (niranjan94) [inline app/templates/gentelella/users/events/sessions/base_session_table.html]: @sudheesh001 could you explain a bit ? That's as per datatables documentation ... AFAIK, that's a right way to increase page length ...

Comment (sudheesh001) [inline app/templates/gentelella/users/events/sessions/base_session_table.html]: @niranjan94 Can you point me to the documentation.

**Pull request merged**